packages/babel-plugin-transform-runtime/scripts/build-dist.js

```diff
@@ -262,3 +285,18 @@ function buildRuntimeRewritePlugin(runtimeName, helperName, esm) {
     },
   };
 }
+
+function addDefaultCJSExport({ template }) {
+  return {
+    visitor: {
+      Program: {
+        exit(path) {
+          path.pushContainer(
+            "body",
+            template.statement.ast`module.exports.default = module.exports`
+          );
```

Comment:
This should probably also add the `__esModule` property.

Reply:
I thought about it, but I cannot think of a case when it's needed (since `module.exports === module exports.default`)

Reply:
Well, not using `__esModule` results in the `_interopRequireDefault` and the `_interopRequireWildcard` helpers creating an extra `{ default: mod }` object.

Reply:
Oh that's true, I'll update it.